pkg/network/ovn_kubernetes.go

```diff
@@ -232,7 +232,7 @@ func boostrapOVN(kubeClient client.Client) (*bootstrap.BootstrapResult, error) {
 
 	controlPlaneReplicaCount, _ := strconv.Atoi(rcD.ControlPlane.Replicas)
 
-	err := wait.PollImmediate(2*time.Second, 60*time.Second, func() (bool, error) {
+	err := wait.PollImmediate(5*time.Second, 280*time.Second, func() (bool, error) {
```

Comment:
out of curiosity,:
1) any particular reason why we also bumped the polling interval to 5secs? - just to reflect the relativity wrt to the timeout interval?
2) do we know approximately how long it takes with the new model?

Reply:
1. Yes
2. No, neither I nor @trozet performed a real study on the time it normally takes, and what we should set these values to....but then again, that was never the case the first time we implemented this mechanism either. With OVS running on the node and attached to the primary NIC it has however been seen that nodes might boot a bit slower due to DHCP issues: https://github.com/openshift/machine-config-operator/pull/1860#issuecomment-653354832.

Reply:
understood, thanks for clarifying Alex.